app/pages/nonsipp/employercontributions/EmployerContributionsSectionStatus.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.employercontributions
+
+import models.SchemeId.Srn
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+import viewmodels.models.SectionStatus
+
+case class EmployerContributionsSectionStatus(srn: Srn) extends QuestionPage[SectionStatus] {
```

Comment:
Struggling to see where is that being set...

Reply:
this will be set when the user says they are done adding contributions on the member list page so the task list can mark it as complete - should save us from individually looking at pages in the journey from the task list 

We can also set it as in progress when the user selects yes on the introductory page on the contributions journey